juniper_warp/src/lib.rs

```diff
@@ -199,39 +200,45 @@ where
 /// let graphql_filter = make_graphql_filter(schema, context_extractor);
 ///
 /// let graphql_endpoint = warp::path("graphql")
-///     .and(warp::post2())
+///     .and(warp::post())
 ///     .and(graphql_filter);
-/// # }
 /// ```
-pub fn make_graphql_filter<Query, Mutation, Context, S>(
-    schema: juniper::RootNode<'static, Query, Mutation, S>,
+pub fn make_graphql_filter<Query, Mutation, Subscription, Context, S>(
```

Comment:
This should have a `_sync` suffix and the other ones with `_async` need it deleted.